### DISCUSSION TITLE
Bookmarks don't open

### DISCUSSION THREAD

**Discussion by alisicki** [dbeaver/dbeaver#18933] [Q&A] [answered]
### Description

https://user-images.githubusercontent.com/44769516/216077528-ed87ad96-03fb-46f2-8712-841a6a34f90a.mp4

Can't open an object using its bookmark. 

### DBeaver Version

Version 22.3.3.202301211053

### Operating System

Windows 10

**Comment by E1izabeth** [comment]
What database do you use?

**Comment by E1izabeth** [comment, marked as answer]
It seems that you switch the perspective and trying to open DBeaver bookmark with eclipse editor.
Try switching it back to DBeaver in the right top corner
![image](https://user-images.githubusercontent.com/28875055/216080126-a2789354-7fd1-49ac-ab9b-22f5db7f4b3b.png)

**Comment by alisicki** [comment]
Thank you for your help. :)
Switching it back to DBeaver helped.
